h2o-py/h2o/estimators/naive_bayes.py

```diff
@@ -788,3 +788,18 @@ def export_checkpoints_dir(self, export_checkpoints_dir):
         self._parms["export_checkpoints_dir"] = export_checkpoints_dir
 
 
+    @property
+    def gainslift_bins(self):
+        """
+        Gains/Lift table bins
+
+        Type: ``int``  (default: ``0``).
```

Comment:
default is 0?

Reply:
Yes. https://github.com/h2oai/h2o-3/pull/4730/files/cbd7de9c2bcb6685d4f40888ab1999ea7c3ba1db#r444702721

Reply:
The original designed revolved around 0 as default, so I only kept it. If unspecified, H2O will "do it's own thing" automatically.

Reply:
My point is that default is not actually 0, by default we have 16 bins that are partitioned using a non-linear scale. Since we are exposing this parameter it should say what the default does in the method documentation. Instead of using 0, I would maybe use -1 (use default 16) and 0 to disable (no bins). What do you think?